### PR TITLE
Add Bahasa Indonesia translation for addons

### PR DESCRIPTION
**Pull request by danninov**
This PR addresses #13929

### CODE REVIEW

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Instalasi Addons
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/addons.md]: Wondering if we should use Add-ons or Addons

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+{{% capture overview %}}
+
+
+*Add-ons* berfungsi untuk menambah serta memperluas fungsionalitas dari Kubernetes.
+
+Laman ini akan menjabarkan beberapa *add-ons* yang tersedia serta tautan instruksi bagaimana cara instalasi masing-masing *add-ons*.
+
+*Add-ons* pada setiap bagian akan diurutkan secara alfabet - pengurutan ini tidak dimaksudkan sebagai status preferensi atau keunggulan.
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/addons.md]: tidak dilakukan berdasarkan sattus prefenrensi atau keunggulan

File: content/id/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (danninov) [reply content/id/docs/concepts/cluster-administration/addons.md]: I prefer Add-ons, easier to read

Comment (irvifa) [reply content/id/docs/concepts/cluster-administration/addons.md]: please fix it then and change it to Add-ons

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+
+{{% capture body %}}
+
+## Jaringan dan *Policy* Jaringan
+
+
+* [ACI](https://www.github.com/noironetworks/aci-containers) menyediakan integrasi jaringan kontainer dan keamanan jaringan dengan Cisco ACI.
+* [Calico](https://docs.projectcalico.org/latest/getting-started/kubernetes/) merupakan penyedia jaringan L3 yang aman dan *policy* jaringan.
+* [Canal](https://github.com/tigera/canal/tree/master/k8s-install) menggabungkan Flannel dan Calico, menyediakan jaringan serta *policy* jaringanp.
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/addons.md]: nit: typo jaringanp

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+* [CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie) memungkinkan Kubernetes agar dapat terkoneksi dengan beragam *plugin* CNI, seperti Calico, Canal, Flannel, Romana, atau Weave dengan mulus.
+* [Contiv](http://contiv.github.io) menyediakan jaringan yang dapat dikonfigurasi (*native* L3 menggunakan BGP, *overlay* menggunakan vxlan, klasik L2, dan Cisco-SDN/ACI) untuk berbagai penggunaan serta *policy framework* yang kaya dan beragam. Proyek Contiv merupakan proyek [open source](http://github.com/contiv). Laman [instalasi](http://github.com/contiv/install) ini akan menjabarkan cara instalasi, baik untuk kluster dengan kubeadm maupun non-kubeadm.
+* [Contrail](http://www.juniper.net/us/en/products-services/sdn/contrail/contrail-networking/), yang berbasis dari [Tungsten Fabric](https://tungsten.io), merupakan sebuah proyek *open source* yang menyediakan virtualisasi jaringan multi-cloud serta platform manajemen *policy*. Contrail dan Tungsten Fabric terintegrasi dengan sistem orkestrasi lainnya seperti Kubernetes, OpenShift, OpenStack dan Mesos, serta menyediakan mode isolasi untuk mesin virtual (VM), kontainer/pod dan *bare metal*.
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/addons.md]: nit: multi-cloud -> _multi-cloud_

File: content/id/docs/concepts/cluster-administration/addons.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* [NSX-T](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) Container Plug-in (NCP) menyediakan integrasi antara VMware NSX-T dan orkestrator kontainer seperti Kubernetes, termasuk juga integrasi antara NSX-T dan platform CaaS/PaaS berbasis kontainer seperti *Pivotal Container Service* (PKS) dan OpenShift.
+* [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) merupakan platform SDN yang menyediakan *policy-based* jaringan antara Kubernetes Pods dan non-Kubernetes *environment* dengan *monitoring* visibilitas dan keamanan.
+* [Romana](http://romana.io) merupakan solusi jaringan  *Layer* 3 untuk jaringan pod yang juga mendukung [*NetworkPolicy* API](/docs/concepts/services-networking/network-policies/). Instalasi Kubeadm *add-on* ini tersedia [di sini](https://github.com/romana/romana/tree/master/containerize).
+* [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) menyediakan jaringan serta *policy* jaringan, yang akan membawa kedua sisi dari partisi jaringan, serta tidak membutuhkan *database* eksternal.
```
Comment (irvifa) [inline content/id/docs/concepts/cluster-administration/addons.md]: nit: *database* -> basis data

**Pull request merged**